backend/src/controller/chatController.ts

```diff
@@ -62,23 +63,31 @@ function createNewUserMessages(
 				infoMessage: messageTransformation.transformedMessageInfo,
 			},
 			{
-				completion: {
-					role: 'user',
-					content: messageTransformation.transformedMessageCombined,
-				},
+				completion:
+					createAs === 'completion'
+						? {
+								role: 'user',
+								content: messageTransformation.transformedMessageCombined,
+						  }
+						: undefined,
 				chatMessageType: 'USER_TRANSFORMED',
 				transformedMessage: messageTransformation.transformedMessage,
 			},
 		];
 	} else {
 		return [
-			{
-				completion: {
-					role: 'user',
-					content: message,
-				},
-				chatMessageType: 'USER',
-			},
+			createAs === 'completion'
+				? {
+						completion: {
```

Comment:
Is indentation squiffy here? One too many indents?

Reply:
just Prettier doing its thing. Looks better in the ide

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/e6058246-d6b1-448d-87b7-3ad94b5777b9)